modules/apps/journal/journal-demo-data-creator-impl/src/main/resources/com/liferay/journal/demo/data/creator/internal/dependencies/article3/content.txt

```diff
@@ -1,4 +1,4 @@
-<p><a title="Photo: Myrabella / Wikimedia Commons, via Wikimedia Commons" href="https://commons.wikimedia.org/wiki/File%3AChamp_de_colza_C%C3%B4te-d'Or_Bourgogne_avril_2014.jpg"><img width="512" alt="Champ de colza Côte-d&#039;Or Bourgogne avril 2014" src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/10/Champ_de_colza_C%C3%B4te-d%27Or_Bourgogne_avril_2014.jpg/512px-Champ_de_colza_C%C3%B4te-d%27Or_Bourgogne_avril_2014.jpg"/></a></p>
+<p><a title="Photo: Myrabella / Wikimedia Commons, via Wikimedia Commons" href="https://commons.wikimedia.org/wiki/File%3AChamp_de_colza_C%C3%B4te-d'Or_Bourgogne_avril_2014.jpg"><img width="512" alt="Champ de colza Côte-d&#039;Or Bourgogne avril 2014" src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/10/Champ_de_colza_C%C3%B4te-d%27Or_Bourgogne_avril_2014.jpg/512px-Champ_de_colza_C%C3%B4te-d%27Or_Bourgogne_avril_2014.jpg"/></a></p>
```

Comment:
Unsure if `0xa0` (non-breaking space) is critical here, to avoid breaking lines.